fluent-resmgr/src/resource_manager.rs

```diff
@@ -92,24 +109,41 @@ impl ResourceManager {
         &self,
         locales: Vec<LanguageIdentifier>,
         resource_ids: Vec<String>,
-    ) -> impl Iterator<Item = FluentBundle<&FluentResource>> {
-        let res_mgr = self;
-        let mut idx = 0;
-
-        iter::from_fn(move || {
-            locales.get(idx).map(|locale| {
-                idx += 1;
-                let mut bundle = FluentBundle::new(vec![locale.clone()]);
-                for resource_id in &resource_ids {
-                    let resource = res_mgr.get_resource(resource_id, &locale.to_string());
-                    bundle.add_resource(resource).unwrap();
-                }
-                bundle
-            })
-        })
+    ) -> Result<impl Iterator<Item = FluentBundle<&FluentResource>>, Vec<ResourceManagerError>>
+    {
+        let mut ptr = 0;
```

Comment:
Nit: `idx` is clearer choice here compared to `ptr`. I think this may have been missed in a rebase.